modules/apps/asset/asset-publisher-web/src/main/java/com/liferay/asset/publisher/web/internal/display/context/AssetPublisherDisplayContext.java

```diff
@@ -2266,10 +2267,18 @@ private String _getSegmentsAnonymousUserId() {
 	}
 
 	private long[] _getSegmentsEntryIds() {
-		return _segmentsEntryRetriever.getSegmentsEntryIds(
-			_themeDisplay.getScopeGroupId(), _themeDisplay.getUserId(),
-			_requestContextMapper.map(
-				PortalUtil.getHttpServletRequest(_portletRequest)));
+		long[] requestSegmentsEntryIds = GetterUtil.getLongValues(
+			_portletRequest.getAttribute(SegmentsWebKeys.SEGMENTS_ENTRY_IDS));
+		long[] portletSegmentsEntryIds =
+			_segmentsEntryRetriever.getSegmentsEntryIds(
+				_themeDisplay.getScopeGroupId(), _themeDisplay.getUserId(),
+				_requestContextMapper.map(
+					PortalUtil.getHttpServletRequest(_portletRequest)));
```

Comment:
@tototrinh @hongvo2308 Thanks for the fix! ❤️ 
LGTM, I have seen only a minor issue, could you please change this call to not use the deprecated method?
Thank you very much!